### PR TITLE
[ADD] Add new integration step

### PR DESCRIPTION
**Pull request by bouskaJ**
Enable to create Camel-K integrations using YAKS.

### CODE REVIEW

File: java/yaks-testing-camel-k/src/main/java/dev/yaks/testing/camel/k/CamelKSteps.java [hunk 1]
```diff
@@ -89,8 +125,80 @@ private Pod getRunningIntegrationPod(String integration) {
...
+		Matcher matcher = pattern.matcher(trait);
+		return matcher.matches();
+	}
+
+
+
     private String namespace() {
-        return System.getenv("NAMESPACE");
+        String result = System.getenv("NAMESPACE");
+		final File namespace = new File("/var/run/secrets/kubernetes.io/serviceaccount/namespace");
```
Comment (christophd) [inline java/yaks-testing-camel-k/src/main/java/dev/yaks/testing/camel/k/CamelKSteps.java]: Can we make this work for non-unix systems (like Windows), too?

**Comment by christophd** [comment]
@nicolaferraro can you also have a look because I am not that familiar with Camel K CRDs and roles

File: java/yaks-testing-camel-k/pom.xml [hunk 2]
```diff
@@ -42,6 +42,17 @@
...
 
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+            <version>${jackson.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
```
[the remark this answers is not in the document]
Comment (bouskaJ) [reply java/yaks-testing-camel-k/pom.xml]: fixed in https://github.com/bouskaJ/yaks/blob/add_new_integration_feature/java/pom.xml#L143

File: java/yaks-testing-camel-k/src/main/java/dev/yaks/testing/camel/k/CamelKSteps.java (hunk 1 above)
Comment (bouskaJ) [reply java/yaks-testing-camel-k/src/main/java/dev/yaks/testing/camel/k/CamelKSteps.java]: The test should be running in the OCP container. The container should be always unix-like system.

**Pull request merged**